tests/_helpers.py

```diff
@@ -368,3 +368,406 @@ def _import_rpy2_mgcv() -> bool:
 gamr_skip = pytest.mark.skipif(
     _import_rpy2_mgcv(), reason="Cannot import `rpy2` or R's `mgcv` package."
 )
+
+
+def test_matrix_1() -> np.ndarray:
```

Comment:
Should not be in `_helpers.py`, should be a `@pytest.fixture(scope='session')` in `conftest.py`.
It will get instantiated only once (i.e. more efficient) - useful since you don't write to it.

Reply:
done.